### PR TITLE
Feature/typeahead search [OSF-6594]

### PR DESCRIPTION
**Pull request by leodomingo**
Implementing a search feature for the application, it currently operates by accepting the user's query as it's typed, appending it as a queryParam on the route and then fetching the resource from the djangoapi search query endpoint. Right now, the search is nearly instantaneous even with upwards of 10 thousand conferences in the database. 

After every search, ember only appends the new unique models that it finds to its browser data and only actually receives 1 page of responses, meaning that the browser won't get bogged down unless the user searches for a large amount of conferences without ever actually pressing the submit button, and even then this is easily rectified by clearing the cache after each search.

### CODE REVIEW

File: meetings/conferences/models.py [hunk 1]
```diff
@@ -3,7 +3,6 @@
 from django.db import models
 from django_countries.fields import CountryField
 
-
```
Comment (Ghalko) [inline meetings/conferences/models.py:8]: https://www.python.org/dev/peps/pep-0008/ (2 lines before classes)

File: app/styles/app.scss [hunk 2]
```diff
@@ -1,14 +1,15 @@
-@import "ember-modal-dialog/ember-modal-structure";
+ @import "ember-modal-dialog/ember-modal-structure";
```
Comment (Ghalko) [inline app/styles/app.scss]: space?

File: app/styles/app.scss [hunk 3]
```diff
@@ -50,18 +51,21 @@ p
 }
 .navbar
 {
-  background-color: transparent;
-  background: transparent;
   border-color: transparent;
-  position: absolute;
+  position: fixed;
```
Comment (jnayak1) [inline app/styles/app.scss:55]: I would say this should be it's own class so it doesn't mess with the navbar that we will be getting from ember-osf eventually.

File: app/styles/index.scss [hunk 4]
```diff
@@ -95,43 +100,59 @@ Splash page typography settings
...
 .masonry-grid
 {
   margin: 0 auto;
 }
+table 
+{
+  color: black;
+  border-collapse: collapse;
+  background-color: white;
+}
```
Comment (jnayak1) [inline app/styles/index.scss:115]: Same with this

**Pull request merged**